python/cudf/cudf/core/dataframe.py

```diff
@@ -3247,6 +3247,99 @@ def drop_duplicates(
 
         return self._mimic_inplace(outdf, inplace=inplace)
 
+    @_cudf_nvtx_annotate
+    def duplicated(self, subset=None, keep="first"):
+        """
+        Return boolean Series denoting duplicate rows.
+
+        Considering certain columns is optional.
+
+        Parameters
+        ----------
+        subset : column label or sequence of labels, optional
+            Only consider certain columns for identifying duplicates, by
+            default use all of the columns.
+        keep : {'first', 'last', False}, default 'first'
+            Determines which duplicates (if any) to mark.
```

Comment:
This sentence is written poorly (in 3 different ways!) in all the upstream docs for DataFrame, Series, and Index. I would prefer this line in all three docstrings:
```suggestion
            Determines the value or values in a set of duplicates to mark.
```

Reply:
Done